randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2143,6 +2143,12 @@ Extra - asset_id: collision_camera_032
           All of the following:
               Gravity Suit
               Simple IBJ or Use Spin Boost
+          All of the following:
+              # Ledge Warp (Advanced Movement) makes this viable https://www.youtube.com/watch?v=ND5zby3zaz8
```

Comment:
Do we have a consensus about which form of links we use? Most of the links use the shortened version but not all.

Reply:
Yeah I noticed that when I added this the line was so long. I can shorten it to make it more readable.